python/generate/templates/configuration.mustache

```diff
@@ -177,7 +177,7 @@ conf = {{{packageName}}}.Configuration(
                  server_index=None, server_variables=None,
                  server_operation_index=None, server_operation_variables=None,
                  ssl_ca_cert=None,
-                 tcp_keep_alive=False,
+                 tcp_keep_alive=False,flag_to_use_decimal=False
```

Comment:
This should be on a new line.

Reply:
On second thought, I'll leave this up to your discretion, can't see anything in PEP8 guidelines about this.